spec/models/miq_event_definition_spec.rb

```diff
@@ -1,5 +1,28 @@
 RSpec.describe MiqEventDefinition do
   let(:event_defs) { MiqEventDefinition.all.group_by(&:name) }
+
+  it "doesn't access database when unchanged model is saved" do
+    m = FactoryBot.create(:miq_event_definition)
+    expect { m.valid? }.not_to make_database_queries
+  end
+
+  describe "name validation" do
+    it "allows nil" do
+      expect { FactoryBot.create(:miq_event_definition) }
```

Comment:
will you pass in a name here. so we know that no name is set from the factory?

I expected the presence check to blow up here